saleor/graphql/order/mutations/order_note_common.py

```diff
@@ -0,0 +1,42 @@
+import graphene
+from django.core.exceptions import ValidationError
+
+from ....order.error_codes import OrderErrorCode
+from ...core.doc_category import DOC_CATEGORY_ORDERS
+from ...core.mutations import BaseMutation
+from ...core.types import BaseInputObjectType
+from ...core.validators import validate_required_string_field
+
+
+class OrderNoteInput(BaseInputObjectType):
+    message = graphene.String(
+        description="Note message.", name="message", required=True
+    )
+
+    class Meta:
+        doc_category = DOC_CATEGORY_ORDERS
+
+
+class OrderNoteCommon(BaseMutation):
```

Comment:
IMO we should name it `OrderNoteMixin` as we using this convention in other places.

Reply:
I was based on `OrderDiscountCommon` actually, which is in the same directory and was added a year ago. As far as I can see it's not consistent and there's `Base` suffix also used quite often, but I can change it to `Mixin` if that's preferred.